packages/docs/src/locales/en/en.json

```diff
@@ -661,16 +661,18 @@
         "canBeFocused": "If false, then input can not be focused using mouse or keyboard."
       },
       "events": {
-        "input": "Emitted when the component needs to change the model",
-        "change": "Emitted when the input is changed by user interaction",
```

Comment:
![image](https://user-images.githubusercontent.com/5394573/147937388-1d12abde-c7fa-438c-b360-486afd52ba54.png)

Manual input should not allow for keyboard navigation in dropdown.
Dropdown should still be visible though (for mouse users).
Might be not the best solution, but probably good enough for now.